abstract_specification.md

```diff
@@ -618,6 +618,10 @@ _Note:_ this API was preceded by another (un-versioned) API draft, which is coll
 ## Indexing and slicing
 
 * In the above `read()` methods, indexing by an empty list of IDs must result in zero-length query results.
+* Negative indices are not to be interpeted as aliases for positive indices (as is common in Python) or as exclusionary (as is common in R).
```

Comment:
Nit: prefer if the wording was more definitive, i.e.,  "Negative indices must not be interpreted as ..."